recipes/oddt/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "oddt" %}
+{% set version = "0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 330f510a6563c371627370200fbfb39ed3496662d067d17e09edb5a6025c8ae6
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - six
+    - numpy >=1.11
+    - scipy >=0.17
+    - scikit-learn >=0.18
+    - scikit-image >=0.12.3
+    - joblib >=0.9.4
```

Comment:
I think you can make this recipe `noarch`, although some dependencies may be missing for Windows, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages.

Reply:
I tried to enable windows but there is a lot of deps conflicts that makes the CI failed.

Reply:
`noarch` packages are built only on Linux, so there shouldn't be any Windows CI failures. Also, I just checked the dependencies and they are available for all platforms, so please go ahead and make this recipe `noarch: python` and remove `skip: true  # [win]`.

Reply:
see https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/31482355

Reply:
As I mentioned earlier, `noarch` recipes are built only on Linux, and we can ignore the Windows failures. When the package is installed on Windows, any missing dependences can be additionally installed there. The advantage of using `noarch` packages is in the CI resources used, so as long as package can be `noarch` it should be.

Reply:
Perfect!